src/main/java/ch/njol/skript/Skript.java

```diff
@@ -542,6 +542,7 @@ public void onEnable() {
 		try {
 			getAddonInstance().loadClasses("ch.njol.skript",
 				"conditions", "effects", "events", "expressions", "entity", "sections", "structures");
+			getAddonInstance().loadClasses("org.skriptlang.skript", "elements");
 		} catch (final Exception e) {
```

Comment:
Confused on why we're using `org.skriptlang.skript.elements` I've noticed a few things use this now and figured I might as well ask when and why should this be used? It seems like a oddity between skript's normal path of stuff

Reply:
We're moving syntaxes to this package. We're currently using it for major collections, like displays and this fireworks package.